deepchem/feat/graph_data.py

```diff
@@ -10,13 +11,13 @@ class GraphData:
 
     Attributes
     ----------
-    node_features: np.ndarray
```

Comment:
You want to describe this using a union type https://mypy.readthedocs.io/en/stable/kinds_of_types.html#union-types

Reply:
It is not a union type. I forgot to remove the torch.tensor from the docstring. It only accepts numpy per our discussion this morning.